wain-syntax-text/src/ast.rs

```diff
@@ -1,6 +1,10 @@
-use std::borrow::Cow;
+use alloc::fmt;
+use alloc::{borrow::Cow, vec::Vec};
+#[cfg(feature = "hashbrown")]
+use hashbrown::HashMap;
```

Comment:
`alloc::collections::BTreeMap` is available. Can we use `BTreeMap` instead of introducing a new dependency `hashbrown` on `no_std`? Or should we still use hashbrown?

https://github.com/rust-lang/rust/blob/80d8270d8488957f62fbf0df7a19dfe596be92ac/library/alloc/src/collections/mod.rs#L39